README.md

```diff
@@ -679,6 +679,61 @@ It also means that these rules will match against following two events :
 
 This behaviour may change in future version (to avoid any confusions) and should not be relied upon.
 
+#### Caveat: JSON parsing and duplicate keys
+
+Events or Rules containing duplicate JSON keys are invalid. When duplicate keys are passed Ruler 
```

Comment:
Not really "invalid" because we accept them, right? Maybe just "considered poorly formed"?